### ISSUE TITLE
Simulation for double editing pulse MegaPress sequence

### ISSUE THREAD

**Issue by bsjeong** [CIC-methods/FID-A#94]
Hi,

With editing "run_simMegPressShaped.m" file, I'd like to simulate basis set for MegaPress sequence using double editing pulse.
My questions are
1) Dual band RF were applied at -1.9 and -4.7 ppm.
In this case, do I add 4.7 as well as 1.9 for "editOnFreq"? Otherwise, 4.7 can be neglected?

2) In edit OFF pulse sequence, the dual band RF pulse is shifted about -3.01 ppm.
What is the correct value for "editOffFreq"? 4.91 (=1.9 + 3.01) or 7.71 (=4.7+3.01)

3) In event block, refocusing pulses with 180 degree were applied at - 4.7 ppm.
If so, refPhCyc1 & 2 are [0, 180] & [0,180] in contrast to [0,90] in default value in the example script.
I believe editPhCyc1 & 2 are [0 90] & [0 90] same as in the example script.

4) If exponential line broadening is 3 Hz, is the linewidth of the output spectrum 3 Hz?

Thanks in advance,

Bumseok

**Comment by jamienear** [comment]
Dear Bumseok,

1.  In order to use a dual-band RF pulse, you will need to change the ‘refocWaveform’ parameter to point to a dual band RF pulse.  By default, ‘refocWaveform’ is set to ’sampleRefocPulse.pta’, which is a single banded gaussian editing pulse, so you will need to change this to a dual band version.

2.  The precise values that you choose for EditOnFreq and EditOffFreq depend on the specifics of your dual band pulse.  Does your RF pulse have one band at zero and one band off resonance?  Or does your RF pulse have two bands that are symmetrically placed off resonance?  What are the exact frequencies of the bands when you simulate your pulse using rf_blochSim?

3.  No the values of refPhCyc are not related to the flip angles.  These are the phases of the RF pulses required for phase cycling.  Please leave these at their default values of [0 90] and [0 90].

4.  Yes, the line width of the resulting spectrum should be approximately equal to the value of the “lw” parameter that you set.

Hope this helps.

Best wishes,
Jamie




On Jul 6, 2022, at 10:30 PM, bsjeong ***@***.******@***.***>> wrote:



Hi,

With editing "run_simMegPressShaped.m" file, I'd like to simulate basis set for MegaPress sequence using double editing pulse.
My questions are

  1.  Dual band RF were applied at -1.9 and -4.7 ppm.
In this case, do I add 4.7 as well as 1.9 for "editOnFreq"? Otherwise, 4.7 can be neglected?

  2.  In edit OFF pulse sequence, the dual band RF pulse is shifted about -3.01 ppm.
What is the correct value for "editOffFreq"? 4.91 (=1.9 + 3.01) or 7.71 (=4.7+3.01)

  3.  In event block, refocusing pulses with 180 degree were applied at - 4.7 ppm.
If so, refPhCyc1 & 2 are [0, 180] & [0,180] in contrast to [0,90] in default value in the example script.
I believe editPhCyc1 & 2 are [0 90] & [0 90] same as in the example script.

  4.  If exponential line broadening is 3 Hz, is the linewidth of the output spectrum 3 Hz?

Thanks in advance,

Bumseok

**Comment by bsjeong** [comment]
Dear Jamie,

Thank you so much for your reply.
I got your answer regarding to my question 3 and 4.

It is hard to follow your answer 1 and 2 as, I believe, I'm a very beginner in terms of MR sequence stuff.
I'm worried that the following question doesn't make any sense at all.

1. How to create a dual band version of gaussian editing pulse?
It seems "rf_dualBand.m" may be the code for it, but I need your help.
Just I guess few parameters for it.

tp (pulse duration, ms) = 19.712 ;
df (frequency of 2nd gaussian band, Hz) = ? ; (2.8ppm = -1.9 + 4.7)  
n = 400; (just follow to it in single band pulse)
bw = ? ;
ph = ? ; (610.28)
shft = ? ; (3.01ppm)


2. if I understand my event blocks in my dual band pulse, the RF pulse have two bands may be symmetrically placed off resonance and exact frequencies of band may be 299 and -209, respectively.
in editing on sequence, 
The 1st refocusing pulse (180 degree of flip, 7ms long) have 299/-286.74 ~ 0/-466.74 (Freq/Phase)
The 1st editing pulse(19.712ms long) is placed between -172/610.28 ~ 0/610.28
The 2nd refocusing pulse (180 degree, 0.6ms long) have -209/112.57 ~ 0/-67.43
The 2nd editing pulse(19.712ms long) is placed between -172/610.28 ~ 0/610.28
The 3rd (180 degree of flip, 7ms long) have -159/ 290.34 ~ 0/110.34

In editing OFF sequence, both frequency and phase of editing pulse is change from -172 to 172. 610.28 to -610.28, respectively.
The 1st editing pulse(19.712ms long) is placed between 172/-610.28 ~ 0/-610.28
The 2nd editing pulse(19.712ms long) is placed between 172/-610.28 ~ 0/-610.28

Warm regards,

Bumseok
